ts/src/test/Exchange/base/test.sharedMethods.ts

```diff
@@ -126,7 +126,11 @@ function assertTimestamp (exchange, skippedProperties, method, entry, nowToCheck
         const dt = entry['datetime'];
         if (dt !== undefined) {
             assert (typeof dt === 'string', '"datetime" key does not have a string value' + logText);
-            assert (dt === exchange.iso8601 (entry['timestamp']), 'datetime is not iso8601 of timestamp' + logText);
+            // there are exceptional cases, like getting microsecond-targeted string '2022-08-08T22:03:19.014680Z', so parsed unified timestamp, which carries only 13 digits (millisecond precision) can not be stringified back to microsecond accuracy, causing the bellow assertion to fail
+            //    assert (dt === exchange.iso8601 (entry['timestamp']))
+            // so, we have to compare with millisecond accururacy
+            const dtParsed = exchange.parse8601 (dt);
+            assert (exchange.iso8601 (dtParsed) === exchange.iso8601 (entry['timestamp']), 'datetime is not iso8601 of timestamp' + logText);
```

Comment:
this will fix some exceptional cases, like `blockchaincom` having microsecond datetime string